src/aspire/image/__init__.py

```diff
@@ -228,7 +229,7 @@ def rotate(self):
     def save(self, mrcs_filepath, overwrite=False):
         with mrcfile.new(mrcs_filepath, overwrite=overwrite) as mrc:
             # original input format (the image index first)
-            mrc.set_data(self.data.astype('float32'))
+            mrc.set_data(self.data.astype(np.float32))
```

Comment:
Just for my own curiosity, why is `np.float32` preferred over `'float32'`? Speed?

Reply:
Generally I always see use of the string names considered legacy. Sometimes they're just poor/ambiguous like in the case of `complex`.  I'd rather give np the np object.

One might also prefer using the strings only for special places in the code for other reasons...

Reply:
Makes sense.